app/components/SearchBox.tsx

```diff
@@ -47,10 +47,18 @@ const SearchBox: React.FunctionComponent<Props> = ({
 
   const uiSchema = {
     searchField: {
-      'ui:col-md': 5
+      'ui:col-md': 5,
+      'ui:placeholder': 'Search By:',
+      'ui:options': {
+        label: false
+      }
     },
     searchValue: {
-      'ui:col-md': 7
+      'ui:col-md': 7,
+      'ui:placeholder': 'Search Term',
+      'ui:options': {
+        label: false
+      }
```

Comment:
Ditto - can you find an alternative to accomplish the goal of alignment without changing the search inputs' labelling UI? (remove placeholder and add label back)